src/wasm2js.h

```diff
@@ -238,8 +247,9 @@ class Wasm2JSBuilder {
 
   // Mangled names cache by interned names.
   // Utilizes the usually reused underlying cstring's pointer as the key.
-  std::unordered_map<const char*, IString> mangledNames[(int)NameScope::Max];
-  std::unordered_set<IString> allMangledNames;
+  std::unordered_map<const char*, IString> mangledNamesMap[(int)NameScope::Max];
+  // Set of all mangled names in a scope.
+  std::unordered_set<IString> mangledNamesSet[(int)NameScope::Max];
```

Comment:
It would be good to use names that describe the purpose rather than the types of these values. Would something like `mangledNamesCache` and `allMangledNames` work?

Reply:
Hmm, those feel less good to me actually. But I see your point. How about `wasmNameToMangledName` for the mapping of wasm names to mangled names, and `mangledNames` for the set of mangled names?